ЛР1

Vladislav04Laptev opened the pull request:
Лаптев В.Ю.

Vladislav04Laptev committed:
ЛР1

heykafka commented on ЛР1/task 1.py:
```diff
@@ -0,0 +1,15 @@
+n = len(numbers) #Кол-во элементов в списке чисел
+
+i = 0
+while (i <= n):
```
Хороший подход, что Вы расписали получение индекса none
Оптимальнее было использовать numbers.index(None)

heykafka commented on ЛР1/task 2.py:
```diff
@@ -0,0 +1,8 @@
+
+n = d // b #целочисленное деление для нахождения числа книг, помещающихся на диск
+
+print("Количество книг, помещающихся на дискету:", int(n))
```
лишнее преобразование в int, // - и так возвращает целочисленное

heykafka commented on ЛР1/task 2.py:
```diff
@@ -0,0 +1,8 @@
+b = 100 * 50 * 25 * 4 #количество байт выделяемое под одну книгу
+d = 1.44 * 1024 * 1024 #объём диска
+
+n = d // b #целочисленное деление для нахождения числа книг, помещающихся на диск
```
название переменных не отражает суть

Vladislav04Laptev committed:
ЛР1 исправления ошибок

В task1 был оптимизирован алгоритм при помощи функции .index(). В task2 были изменены названия переменных на отражающие их суть.

Vladislav04Laptev committed:
Merge pull request #2 from Vladislav04Laptev/Vladislav04Laptev-patch-2

ЛР1 исправления ошибок
